eva_submission/eload_submission.py

```diff
@@ -63,6 +64,13 @@ def _get_dir(self, key):
     def now(self):
         return datetime.now()
 
+    def create_log_file(self):
+        logfile_name = os.path.join(self.eload_dir, "submission.log")
```

Comment:
Can you add the eload number in the file name? Just in can the log file gets moved.

Reply:
Sure..I wil ldo that